.github/workflows/ci-test-backend.yml

```diff
@@ -60,8 +55,11 @@ jobs:
         run: $GITHUB_WORKSPACE/golangci-lint run --config ${GITHUB_WORKSPACE}/backend/.golangci.yml ./...
         working-directory: backend/_example/memory_store
 
-      - name: submit coverage
-        run: $(go env GOPATH)/bin/goveralls -service="github" -coverprofile=$GITHUB_WORKSPACE/profile.cov
-        working-directory: backend
+      - name: Send coverage
+        uses: shogo82148/actions-goveralls@v1
+        with:
+          path-to-profile: ${{ github.workspace }}/profile.cov
+          parallel: true
+          working-directory: backend
         env:
-          COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GOPATH: ${{ github.workspace }}
```

Comment:
Why do we need GOPATH here? It's [marked as legacy](https://github.com/shogo82148/actions-goveralls#use-with-legacy-gopath-mode) in this action doc.

Reply:
Removed